mentat/broadcast.py

```diff
@@ -99,9 +99,7 @@ def connect(self) -> None:
         self._listener_task = asyncio.create_task(self._listener())
 
     def disconnect(self) -> None:
-        if self._listener_task.done():
-            self._listener_task.result()
-        else:
```

Comment:
Why this change? I'm pretty sure we want to keep this since `.result()` will [propagate Exceptions](https://docs.python.org/3.10/library/asyncio-task.html#asyncio.Task.result) correctly.

Reply:
Oh hmm I didn't think about exceptions. I changed it because it throws a CancelledError (since it's pretty much always cancelled at this point); I think I'll just wrap it in a try except for CancelledError and keep it like it was. Thanks!